InteractiveHtmlBom/web/util.js

```diff
@@ -477,6 +478,13 @@ function initDefaults() {
   settings.checkboxes = bomCheckboxes.split(",").filter((e) => e);
   document.getElementById("bomCheckboxes").value = bomCheckboxes;
 
+  settings.darkenWhenChecked = readStorage("darkenWhenChecked") || "";
+  populateDarkenWhenCheckedOptions();
+  var optionToSelect = document.getElementById("darkenWhenChecked-" + settings.darkenWhenChecked);
+  if (optionToSelect) {
+    optionToSelect.checked = true;
+  }
```

Comment:
This needs to be inside populateDarkenWhenCheckedOptions() so that when checkboxes list is updated the selected option is maintained if possible.

Reply:
Done.